hw/ip/otbn/dv/otbnsim/sim/wsr.py

```diff
@@ -82,8 +82,10 @@ def __init__(self, name: str):
         # For now, the RTL doesn't have a real "random number generator".
         # Eventually, it will have an LFSR of some sort, seeded by the
         # CSRNG/EDN. We'll model that properly when we've specced it out. Until
-        # then, random numbers are all 0x1.
-        self._random_value = 1
+        # then, random numbers are constant.  This constant must match the one
+        # in the RTL (the `rnd` signal in the `otbn_core` module found in
+        # rtl/otbn_core.sv). If changed here it must be changed there to match.
+        self._random_value = 0x9999999999999999999999999999999999999999999999999999999999999999
```

Comment:
Looks like the python3 version available under Ubuntu 16.04 LTS doesn't like the syntax with _ so switched to using an unbroken number.

Reply:
This is not an ok literal to have in the code. It's waay too long (how many digits does it have?). If you really must have this value, please construct it with something like this:
```python
u32 = 0x99999999
u64 = (u32 << 32) | u32
u128 = (u64 << 64) | u64
self._random_value = (u128 << 128) | u128
```